prereise/gather/demanddata/transportation_electrification/tests/test_immediate_charging_integration.py

```diff
@@ -42,7 +153,7 @@ def test_immediate_charging_region1():
 
     daily_values = generate_daily_weighting(2017)
 
-    final_result = adjust_bev(
+    adjust_bev(
```

Comment:
Now we just run it without checking the results?

Reply:
Same question below. If we keep these tests for integration test purpose given "unit" tests are added above, should we mark it as integration test with `@pytest.mark.integration`?